src/conn/conn_prefetch.c

```diff
@@ -0,0 +1,176 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "wt_internal.h"
+
+/*
+ * __wt_prefetch_create --
+ *     Start the pre-fetch server.
+ */
+int
+__wt_prefetch_create(WT_SESSION_IMPL *session, const char *cfg[])
```

Comment:
Will the pre-fetch server/utility simply run and start up if prefetch is enabled on a connection-level? How will this work once we shift to having a session-level prefetch configuration as well as a connection-level one?

Reply:
Sounds good, we can address this in WT-11691 and I'll add a FIXME comment as well.